src/ScriptBuilder/Saga/Writers/OracleCorrelationPropertyTypeConverter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace NServiceBus.Persistence.Sql.ScriptBuilder
+{
+    /// <summary>
+    /// Not for public use.
+    /// </summary>
+    public static class OracleCorrelationPropertyTypeConverter
+    {
+        public static string GetColumnType(CorrelationPropertyType propertyType)
+        {
+            switch (propertyType)
+            {
+                case CorrelationPropertyType.DateTime:
+                    return "TIMESTAMP";
+                case CorrelationPropertyType.String:
+                    return "NVARCHAR2(200)";
```

Comment:
Same as MySQL and MSSQL, and I believe inherited from NHibernate. Some rational limit on how long an OrderId (or whatever the correlation property would be) has to be set. Will be documented by the Oracle Scripts page.